pom.xml

```diff
@@ -62,7 +63,7 @@
     <beeju.version>1.2.1</beeju.version>
     <hamcrest-library.version>1.3</hamcrest-library.version>
     <housekeeping.version>1.0.2</housekeeping.version>
```

Comment:
So where is Guava 20.0 being pulled in for Google Cloud?

Reply:
I've downgraded guava all tests run? are we going to need guava-17 for tests and guava-20 for actual deployment?

Reply:
@courtsVII so went back to 20.0 and override that version just in the test. I think we should be fine now as when we package it all up we relocate guava ourselves.

Reply:
Looks good, approving